README.md

```diff
@@ -124,6 +124,12 @@ repo](https://mesonbuild.com/Quick-guide.html#installation-from-source) and call
 - `libelf`, `libz`, `libzstd` if linking against staic `libbpf.a`
 - `bpftool` By default this is cloned and built as part of the default build process. Alternatively it's usually available in `linux-tools-common`.
 
+The kernel has to be built with the following configuration:
+- `CONFIG_BPF=y`
+- `CONFIG_BPF_SYSCALL=y`
+- `CONFIG_BPF_JIT=y`
+- `CONFIG_BPF_LSM=y`
+
```

Comment:
Should we add also:
```
CONFIG_DEBUG_INFO_BTF=y
CONFIG_SCHED_CLASS_EXT=y
```

Also, do we really need BPF_LSM?

Reply:
You're right, it's actually BPF_EVENTS. I also added FTRACE=y.